core/arch/arm/plat-k3/conf.mk

```diff
@@ -0,0 +1,36 @@
+CFG_WITH_STACK_CANARIES ?= y
+CFG_WITH_STATS ?= y
+CFG_CRYPTO_WITH_CE ?= n
+CFG_WITH_SOFTWARE_PRNG ?= y
+CFG_SECURE_DATA_PATH ?= y
+
+ifeq ($(CFG_SECURE_DATA_PATH),y)
+CFG_TEE_SDP_MEM_SIZE ?= 0x00400000
+else
+CFG_TEE_SDP_MEM_SIZE ?= 0x0
+endif
+
+$(call force,CFG_TEE_CORE_NB_CORE,8)
+$(call force,CFG_8250_UART,y)
+$(call force,CFG_GENERIC_BOOT,y)
+$(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
+$(call force,CFG_PM_STUBS,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_ARM_GICV3,y)
+$(call force,CFG_CORE_CLUSTER_SHIFT,1)
+
+ta-targets = ta_arm32
+
+ifeq ($(CFG_ARM64_core),y)
+$(call force,CFG_WITH_LPAE,y)
+ta-targets += ta_arm64
+else
+$(call force,CFG_ARM32_core,y)
+endif
+
+include core/arch/arm/cpu/cortex-armv8-0.mk
+
+arm32-platform-cflags += -Wno-error=cast-align
```

Comment:
Why is this needed? Other ARMv8-A platforms seems to manage without it.

Reply:
This must have been left over from some debug build, it does not seem needed anymore, will remove.